docs/reference/aggregations/bucket/composite-aggregation.asciidoc

```diff
@@ -224,7 +224,7 @@ Time values can also be specified via abbreviations supported by <<time-units,ti
 Note that fractional time values are not supported, but you can address this by shifting to another
 time unit (e.g., `1.5h` could instead be specified as `90m`).
 
-====== Format
+===== Format
```

Comment:
I feel like this section is supposed to be a subsection of the Date Histogram section though. Same with the one below it.

Reply:
You're absolutely right; it should be. But the software is set up to only allow so many heading levels, and there's nothing I can do to change that. As it is, it's nested too far down to appear in the TOC, so the exact level doesn't much matter. If you prefer, I can make it a normal paragraph with bold text so it will look more like the heading level it should be.

Reply:
Maybe a definition list would work too?

Reply:
I'm not really sure how to format it. I'd try just removing the heading all together and see how it reads. Bold text'd be fine with me too. Definition list feels a little lick a hack but, sometimes a hack is the right way to go. I'm not sure!

Reply:
Thanks for the input. Going with the simple solution - bold text (composite-aggregation.asciidoc)